packages/components/src/components/Heading/index.ts

```diff
@@ -35,9 +36,16 @@ export const HeadingComponentDefinition: ComponentDefinition = {
       displayName: 'Height',
```

Comment:
I'm not entirely sure it makes sense to add "Height" css rules to a text element :thinking: